### DISCUSSION TITLE
Problems Downloading Templates on Ollama Web UI

### DISCUSSION THREAD

**Discussion by RaulKong898** [open-webui/open-webui#3845] [General]
Hello,

I encountered a problem using Ollama Web UI. Every time I try to download models I get a server error and can't complete the download. Because of this, I cannot use Ollama Web UI properly.

I would appreciate it if you could help me solve this problem as soon as possible.

Thank you in advance!

Respectfully,
Raul Popescu  

Using Pinokio AI

[logs.zip](https://github.com/user-attachments/files/16200707/logs.zip)

**Comment by oCroson** [comment]
same here!

**Comment by justinh-rahb** [comment]
> Using Pinokio AI

Use something else, like Docker.
